compiler/damlc/tests/daml-test-files/InterfaceEnumViewtype.daml

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR expected monomorphic record type in view type, but found a enum type:
```

Comment:
minor suggestion,

```suggestion
-- @ERROR expected monomorphic record type in view type, but found an enum type:
```

this should be a matter of moving the article into `actuallyFound`

Reply:
done